packages/sn-controls-react/test/QuillOEmbedModule.test.ts

```diff
@@ -0,0 +1,143 @@
+import Quill from 'quill'
+import QuillOEmbedModule from '../src/fieldcontrols/RichTextEditor/QuillOEmbedModule'
+
+const richHtml = '<a href="https://example.com">Example</a>'
+const imageUrl = 'https://example.com'
+const oEmbedUrl = 'https://example.com/oembed?url=http%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DiwGFalTRHDA'
+
+function getFetchResult(type: string) {
+  return () =>
+    Promise.resolve({
+      json: () =>
+        Promise.resolve({
+          type,
+          html: richHtml,
+          url: imageUrl,
+        }),
+    })
+}
+
+Quill.register('modules/oembed', QuillOEmbedModule)
+
+describe('QuillOEmbedModule', () => {
+  const quill = new Quill('div', {
```

Comment:
Because I wasn't sure how to properly include the polyfills, I copied them into a polyfills folder in the tests and added a ts file that allows usage of them. I hope that is at least close to what you meant; the tests seem to run well now.

The polyfills throw huge amounts of linting errors but as they are not checked in the linting process, I didn't bother going through there trying to fix everything :)